Sources/FigmaExport/Loaders/ColorsLoader.swift

```diff
@@ -37,29 +37,39 @@ final class ColorsLoader {
                                                                       lightHC: [Color]?,
                                                                       darkHC: [Color]?) {
         let colors = try loadColors(fileId: figmaParams.lightFileId, filter: filter)
-        
-        let darkSuffix = colorParams?.darkModeSuffix ?? "_dark"
+
+        let lightPrefix = colorParams?.lightModePrefix
+        let darkPrefix = colorParams?.darkModePrefix
+        let lightSuffix = colorParams?.lightModeSuffix
+        let darkSuffix = colorParams?.darkModeSuffix
+        let lightHCPrefix = colorParams?.lightHCModePrefix ?? "lightHC_"
+        let darkHCPrefix = colorParams?.darkHCModePrefix ?? "darkHC_"
         let lightHCSuffix = colorParams?.lightHCModeSuffix ?? "_lightHC"
         let darkHCSuffix = colorParams?.darkHCModeSuffix ?? "_darkHC"
 
-        let lightColors = colors
-            .filter {
-                !$0.name.hasSuffix(darkSuffix) &&
-                !$0.name.hasSuffix(lightHCSuffix) &&
-                !$0.name.hasSuffix(darkHCSuffix)
-            }
```

Comment:
> If you would like to specify light and dark colors in the same file, you can do so with the useSingleFile configuration option. You can then denote dark mode colors by adding a suffix like _dark.
_(from https://github.com/RedMadRobot/figma-export?tab=readme-ov-file#design-requirements)_

You must not delete this code. A color style list can contains colors which names doesn't contain prefixes and suffixes for light mode but contain "dark" suffix/prefix for dark mode. Example: "background", "foreground", "dark_background", "dark_foreground".